src/main/java/Duke.java

```diff
@@ -1,10 +1,380 @@
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Scanner;
+import java.io.File;
+import java.io.FileNotFoundException;
+
+
 public class Duke {
-    public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+
+    //taskList variable for level-2 onwards
+    private static ArrayList<Task> taskList = new ArrayList<Task>();
+    //task count
+    private static int taskCount = 0;
+
+    private static String folderPath = "./data";
+    private static String filePath = folderPath+ "/duke.txt";
+
+
+    //echo function for level-1
+    public static void echo(String message) {
+        System.out.println(message);
+    }
+
+
+    //add function for level-2
+    public static void add(String task) {
```

Comment:
Method names can be more descriptive; what are we adding in this method?

This is a running issue in the entire codebase.